pkg/infra/log/log.go

```diff
@@ -161,7 +161,9 @@ func ReadLoggingConfig(modes []string, logsPath string, cfg *ini.File) error {
 			fileHandler.Maxdays = sec.Key("max_days").MustInt64(7)
 			if err := fileHandler.Init(); err != nil {
 				Root.Error("Failed to initialize file handler", "dpath", dpath, "err", err)
-				return errutil.Wrapf(err, "failed to initialize file handler")
+				// uncomment to crash when it can't create a log file
```

Comment:
we probably want to do the same on few lines above when create the directory
```
			if err := os.MkdirAll(dpath, os.ModePerm); err != nil {
				Root.Error("Failed to create directory", "dpath", dpath, "err", err)
				return errutil.Wrapf(err, "failed to create log directory %q", dpath)
			}
```
to continue instead returning error to keep the behavior consistent

Reply:
I've added this.